main.md

```diff
@@ -579,30 +582,33 @@ This document introduces the following client registration metadata
 DPoP when requesting tokens from the authorization server.
 
 `dpop_bound_access_tokens`
-:   Boolean value specifying whether the client always uses DPoP for token requests.  If omitted, the default value is `false`.
+:   Boolean value specifying whether the client is required to issue DPoP token requests.  If omitted, the default value is `false`.
+
+If `true`, the authorization server MUST reject non-DPoP token requests from this client.
 
-If `true`, the authorization server MUST reject token requests from this client that do not contain the DPoP header.
 
 # Public Key Confirmation {#Confirmation}
 
 Resource servers MUST be able to reliably identify whether
-an access token is bound using DPoP and ascertain sufficient information
-about the public key to which the token is bound in order to verify the
-binding with respect to the presented DPoP proof (see (#http-auth-scheme)). 
+an access token is DPoP-bound and ascertain sufficient information
+to verify the binding of the public key to the DPoP proof (see (#http-auth-scheme)). 
 Such a binding is accomplished by associating the public key 
 with the token in a way that can be
 accessed by the protected resource, such as embedding the JWK
 hash in the issued access token directly, using the syntax described
 in (#jwk-thumb-jwt), or through token introspection as described in
-(#jwk-thumb-intro). Other methods of associating a
-public key with an access token are possible, per agreement by the
+(#jwk-thumb-intro).
+Other methods of associating a
+public key with an access token can be agreed by the
```

Comment:
Please restore the "are possible" wording, as it can be clearly understood by readers.